### PR TITLE
Identity&order in values returned by some APIs

### PR DESCRIPTION
**Pull request by mtrofin**
Clarified that Module.imports, exports, and customSections return new Arrays. 
Clarified order in imports/exports

### CODE REVIEW

File: JS.md [hunk 1]
```diff
@@ -240,11 +240,11 @@ is thrown.
...
 
-This function returns an `Array` produced by mapping each
+This function returns a new `Array` every time it is called. Each such `Array` is produced by mapping each
 [custom section](BinaryEncoding.md#high-level-structure) (i.e., section with
 `id` 0) whose `name` field ([decoded as UTF-8](Web.md#names)) is equal to
 `sectionNameString` to an `ArrayBuffer` containing a copy of the section's
-`payload_data`. (Note: `payload_data` does not include `name` or `name_len`.)
+`payload_data`. (Note: `payload_data` does not include `name` or `name_len`.).
 
 The `Array` is populated in the same order as that in which custom sections
```
Comment (jfbastien) [inline JS.md]: We should use the same wording everywhere to enforce binary-order in the array.

**Comment by mtrofin** [comment]
Incorporated @jfbastien's feedback

File: JS.md [hunk 2]
```diff
@@ -199,14 +199,16 @@ Array exports(moduleObject)
...
-This function returns an `Array` produced by mapping each
+This function returns a new `Array` every time it is called. Each such `Array` is produced by mapping each
 [`Ast.export`](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/ast.ml#L152)
 `e` of [moduleObject.[[Module]].exports](https://github.com/WebAssembly/spec/blob/master/interpreter/spec/ast.ml#L187)
 to the Object `{ name: String(e.name), kind: e.ekind }` where `e.name` is [decoded as UTF8](Web.md#names)
 and `e.ekind` is mapped to one of the String values `"function"`, `"table"`, `"memory"`, `"global"`.
 
 Note: other fields like `signature` may be added in the future.
 
+The returned `Array` is populated in the same order as that in which exports appeared in the WebAssembly binary's exports table.
```
Comment (rossberg-chromium) [inline JS.md]: Nit: simplify sentence to "... is populated in the same order in which exports appear ..." (Also note present tense.)

File: JS.md [hunk 3]
```diff
@@ -227,6 +229,8 @@ to the Object `{ module: String(i.module_name), name: String(i.item_name), kind:
 
 Note: other fields like `signature` may be added in the future.
 
+The returned `Array` is populated in the same order as that in which imports appeared in the WebAssembly binary's imports table.
```
Comment (rossberg-chromium) [inline JS.md]: Same here.

**Comment by mtrofin** [comment]
Incorporated @rossberg-chromium's feedback.

**Pull request merged**